soccer/gameplay/GameplayModule.cpp

```diff
@@ -255,6 +256,27 @@ void Gameplay::GameplayModule::savePlaybook(const string& playbookFile,
     PyGILState_Release(state);
 }
 
+void Gameplay::GameplayModule::clearPlays() {
+    PyGILState_STATE state = PyGILState_Ensure();
+    getMainModule().attr("clear")();
+    PyGILState_Release(state);
+}
+
+bool Gameplay::GameplayModule::checkPlayStatus() {
```

Comment:
I feel like this function could be named something that's more specific to what it does, like `playStatusChanged`?